backend/controllers/iepDetails.js

```diff
@@ -74,8 +74,55 @@ const getIepDetailsFactory = elite2Api => {
     }
   }
 
+  const iconForDifference = {
+    '1': 'Green_arrow.png',
+    '2': 'Double_green_arrow.png',
+    '3': 'TripleGreenArrow.png',
+    '-1': 'Red_arrow.png',
+    '-2': 'Double_red_arrow.png',
+  }
+
+  const levelToIntMap = {
+    Entry: 1,
+    Basic: 2,
+    Standard: 3,
+    Enhanced: 4,
+  }
+
+  const getPossibleLevels = async (context, currentIepLevel, agencyId) => {
+    const levels = await elite2Api.getAgencyIepLevels(context, agencyId)
+
+    return levels
+      .filter(level => level.iepDescription !== currentIepLevel)
+      .filter(level => level.iepDescription !== 'Entry')
+      .map(level => {
+        let diff
```

Comment:
Might look neater to place the diffCalculator in its own method.

Reply:
Definitely looks neater, have abstracted.